app/controllers/players_controller.rb

```diff
@@ -0,0 +1,17 @@
+class PlayersController < ApplicationController
+  def create
+    redirect_to new_user_session_path unless user_signed_in?
+
+    @instance = Instance.find_by_pin(params[:pin])
+    if @instance.blank?
+      redirect_to root_path, notice: "Lobby not found"
+    else # Need some logic to not create dupplicate users (will implement later)
```

Comment:
While testing I realize that we have no logic to prevent duplicate players - I think we can add validations in model and some code here, but that will be for another PR!

Reply:
Sounds good to me, created this Trello so we don't forget about this: https://trello.com/c/ZsoGGTzt/23-dbi-prevent-the-same-player-to-be-created-again